caffe2/python/pybind_state_gpu.cc

```diff
@@ -77,37 +77,29 @@ void addCUDAGlobalMethods(py::module& m) {
       });
   m.def(
       "transform_trt",
-      [](const py::bytes& init_net_str,
-         const py::bytes& pred_net_str,
+      [](const py::bytes& pred_net_str,
```

Comment:
I'm OK with this. Maybe it's a little better to change such a long lambda to a separate function in namespace {} ?

Reply:
Normally, long lambda should be avoided. However, in this case, we are registering a python method. The lambda is just a wrapper as it's more like a function declaration already. So I think it's OK to have it like this here, conforming to what we have been doing to this `pybind_state.cc`. :)